MHCollapsibleViewManager.m

```diff
@@ -212,7 +298,7 @@ -(MHTableViewCell*)returnCellWithIndex:(NSIndexPath*)indexPath tableView:(UITabl
         type = CRUCellViewInteractionHeader;
         collapsed = self.expanded;
         cellText = self.headerTitle;
-        detailText = [NSString stringWithFormat:NSLocalizedString(@"%i items",), self.filterSections.count];
```

Comment:
You can check for english but what I would recommend is having a method that returns the plural of an identifier. e.g. 
```
- (NSString *)identifierForDisplay {
    return NSLocalizedString(self.identifier,);
}

- (NSString *)pluralIdentifierForDisplay {
    NSString *keyForString = [self.identifier stringByAppendingString:@"_plural"];
    return NSLocalizedString(keyForString,);
}
```
.string file
```
"survey" = "survey";
"item" = "item";
"survey_plural" = "surveys";
"item_plural" = "items";
```
Note: I haven't had to do this before so I don't know if this is the best way to do it. I am just trying to make sure everything can be translated. Programmatically making something plural can't be translated.